pages/index.tsx

```diff
@@ -23,6 +24,7 @@ import {
   identifyShift,
   toDate,
   toTime,
+  shmirot
```

Comment:
Please rename to something in english - `positions`? `posts`?

Reply:
done